pkgs/development/python-modules/black/default.nix

```diff
@@ -21,15 +24,17 @@
 
 buildPythonPackage rec {
   pname = "black";
-  version = "22.8.0";
+  version = "22.10.0";
 
```

Comment:
```suggestion
  format = "pyproject";
```

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/194899/commits/2b0bd6c93f83e483441c4a79d8980a7d2fe38105